scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/db/SchedulerDBManager.java

```diff
@@ -441,150 +417,103 @@ public long getRunningJobsCount() {
-                Collection<TaskStatus> taskStatus = Arrays.asList(TaskStatus.SUBMITTED,
-                                                                  TaskStatus.PAUSED,
-                                                                  TaskStatus.PENDING,
-                                                                  TaskStatus.WAITING_ON_ERROR,
-                                                                  TaskStatus.WAITING_ON_FAILURE);
-                Query query = session.getNamedQuery("getPendingTasksCount")
-                                     .setParameterList("jobStatus", NOT_FINISHED_JOB_STATUSES)
-                                     .setParameterList("taskStatus", taskStatus);
-
-                return (Long) query.uniqueResult();
-            }
-
+        return executeReadOnlyTransaction(session -> {
+            Collection<TaskStatus> taskStatus = Arrays.asList(TaskStatus.SUBMITTED,
+                                                              TaskStatus.PAUSED,
+                                                              TaskStatus.PENDING,
+                                                              TaskStatus.WAITING_ON_ERROR,
+                                                              TaskStatus.WAITING_ON_FAILURE);
+            Query query = session.getNamedQuery("getPendingTasksCount")
+                                 .setParameterList("jobStatus", NOT_FINISHED_JOB_STATUSES)
+                                 .setParameterList("taskStatus", taskStatus);
+
+            return (Long) query.uniqueResult();
         });
     }
 
     public long getRunningTasksCount() {
-        return executeReadOnlyTransaction(new SessionWork<Long>() {
-
-            @Override
-            public Long doInTransaction(Session session) {
-                Query query = session.getNamedQuery("getRunningTasksCount")
-                                     .setParameterList("jobStatus", NOT_FINISHED_JOB_STATUSES)
-                                     .setParameterList("taskStatus", Arrays.asList(TaskStatus.RUNNING));
-
-                return (Long) query.uniqueResult();
-            }
+        return executeReadOnlyTransaction(session -> {
+            Query query = session.getNamedQuery("getRunningTasksCount")
+                                 .setParameterList("jobStatus", NOT_FINISHED_JOB_STATUSES)
+                                 .setParameterList("taskStatus", Collections.singletonList(TaskStatus.RUNNING));
 
+            return (Long) query.uniqueResult();
         });
     }
 
     public long getTotalTasksCount() {
-        return executeReadOnlyTransaction(new SessionWork<Long>() {
-
-            @Override
-            public Long doInTransaction(Session session) {
-                Query query = session.getNamedQuery("getTotalTasksCount");
-                Long count = (Long) query.uniqueResult();
-                return count;
-            }
-
+        return executeReadOnlyTransaction(session -> {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove useless curly braces around statement and then remove useless return keyword [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1602)